phoenix-core/src/it/java/org/apache/phoenix/end2end/index/GlobalIndexOptimizationIT.java

```diff
@@ -79,7 +79,8 @@ public void testIndexDeleteOptimizationWithLocalIndex() throws Exception {
             ResultSet rs = conn1.createStatement().executeQuery("EXPLAIN "+ query);
             String expected =
                     "DELETE ROWS CLIENT SELECT\n" +
-                    "CLIENT PARALLEL 1-WAY RANGE SCAN OVER " + dataTableName +" [1,*] - [1,100]\n" +
+                    "CLIENT PARALLEL 1-WAY RANGE SCAN OVER " +
+                            indexTableName+"L" + "(" + dataTableName + ") [1,*] - [1,100]\n" +
```

Comment:
why is "L" appended to the indexTableName here?

Reply:
Because a local index named indexTableName+"L" is created above
(Line 69: `createIndex(indexTableName+"L", dataTableName, "k3", null, true);`)